autofaiss/indices/distributed.py

```diff
@@ -55,7 +55,7 @@ def _yield_embeddings_batch(
 
 
 def _generate_small_index_file_name(batch_id: int) -> str:
-    return f"index_{batch_id}"
+    return "index_" + str(batch_id).zfill(10)
```

Comment:
@rom1504 Note that we can also pass nb_batch to the function to have the exact value like we do in the n-indice branch. But 10 is big enough maybe. WDYT ?

Reply:
I agree it's likely big enough but also think it would be cleaner to have tue right numbers so the file names are a bit shorter (less unnecessary zeros)

Reply:
yeah, let's do it in a propre way then.